utils/init-google-map.ts

```
@@ -1,81 +1,20 @@
-};
-
-let map: google.maps.Map, marker: google.maps.Marker;
-
-export const initGoogleMap = async (probe: Probe, showPulse: boolean = false, markerHasIW: boolean = true, mapCenterYOffsetPx: number | null = null) => {
-	if (!probe) {
-		return;
-	}
+let map: google.maps.Map, marker: google.maps.marker.AdvancedMarkerElement, infoWindow: google.maps.InfoWindow | null;
 
+const createMap = async (element: HTMLElement, center: google.maps.LatLngLiteral, zoom: number) => {
 	const { Map } = await google.maps.importLibrary('maps') as google.maps.MapsLibrary;
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)